app/services/curation_concerns/curation_concern.rb

```diff
@@ -1,9 +1,20 @@
 module CurationConcerns
   module CurationConcern
-    def self.actor(curation_concern, *args)
+    # Returns the top-level actor on the stack
+    def self.actor(curation_concern, current_user, attributes)
+      AddToCollectionActor.new(curation_concern, current_user, attributes,
```

Comment:
This is actually a stack, forgive me for calling it a pipeline.  In the first pass ("down" if you will) in `initialize`, each node may strip off some of the attributes, and on the way up ("create/update") modifies the curation_concern.  I'm not sure how we can do that in a true pipeline.